WMF Framework/WMFOnThisDayEventsFetcher.m

```diff
@@ -0,0 +1,65 @@
+#import "WMFOnThisDayEventsFetcher.h"
+#import "WMFFeedOnThisDayEvent.h"
+#import <WMF/WMF-Swift.h>
```

Comment:
The new syntax for this is `@import  WMF.Swift;` - apparently the old syntax is equivalent to this anyway, but trying to move everything in the app to the new syntax.